src/registrar/tests/common.py

```diff
@@ -238,6 +238,52 @@ def get_table_delete_confirmation_page(self, selected_across: str, index: str):
         )
         return response
 
+    @staticmethod
+    def switchToEnterpriseMode_wrapper(role=UserPortfolioRoleChoices.ORGANIZATION_ADMIN):
```

Comment:
I wonder about mixing cases here. It looks like we otherwise use snake case elsewhere.
```suggestion
    def switch_to_enterprise_mode_wrapper(role=UserPortfolioRoleChoices.ORGANIZATION_ADMIN):
```